client/faucet/src/mint.rs

```diff
@@ -213,11 +213,7 @@ impl Service {
             .as_ref()
             .ok_or_else(|| anyhow::format_err!("get receiver account response not found"))?
             .as_ref();
-        let receiver_seq_num = if let Some(account) = receiver {
-            Some(account.sequence_number)
-        } else {
-            None
-        };
+        let receiver_seq_num = receiver.map(|account| account.sequence_number);
```

Comment:
I prefer the `map`, but last time we made these kinds of lint changes, people complained, so I suggest asking for feedback first.